code/render-modifiers/render-modifiers.handlebars

```diff
@@ -0,0 +1 @@
+ember install @ember/render-modifiers
```

Comment:
If you rename this file to end in `.shell` instead of `.handlebars`, it will have better syntax highlighting.

Reply:
so renaming it to .shell didn't make any syntax highlighting changes but when I renamed it to .js this is what the syntax looks like in the browser - a little prettier?

![Screenshot 2019-11-14 11 53 11](https://user-images.githubusercontent.com/7906064/68891322-6cb7b180-06d5-11ea-9d59-1b24a1ae2b8d.png)

Reply:
ok one more comment - I wonder if it would make better sense to include the dollar sign in the handlebars template?
![Screenshot 2019-11-14 11 58 04](https://user-images.githubusercontent.com/7906064/68891649-20b93c80-06d6-11ea-9b0f-a6c158abd309.png)

Reply:
I have seen people use `$` a lot but I tend not to. The reason is that when I was learning to code, I copy-pasted things as-is and the dollar sign (and the errors it causes) confused me.

Reply:
Ah okay fair enough, makes sense to me. So, I'm thinking to just leave that code snippet as-is (sans the .shell syntax)?  Also, with that said, do you think it would be helpful to include some verbiage so that it's more explicit - something like "in your terminal enter the following command to install @ember/render-modifiers package:"?  I wasn't sure how short we wanted to keep the cheat-sheet.